test/libyul/YulOptimizerTestCommon.cpp

```diff
@@ -356,6 +357,10 @@ YulOptimizerTestCommon::YulOptimizerTestCommon(
 			FakeUnreachableGenerator fakeUnreachableGenerator;
 			fakeUnreachableGenerator(*m_ast);
 			StackLimitEvader::run(*m_context, *m_object, fakeUnreachableGenerator.fakeUnreachables);
+		}},
+		{"commonSwitchCasePrefixMover", [&] () {
+			disambiguate();
```

Comment:
Add function hoister since it is a pre-requisite? Because otherwise fuzzer may find bugs (due to FunctionHoister not being run) which turn out to be false positives.

Reply:
Added it!